tests/yam/validate/validate_copy_network.pm

```diff
@@ -0,0 +1,25 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Validate copy network.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    script_output("ip address show");
+    # The copy is enabled by default.
+    my ($copy_network) = get_var('EXTRABOOTPARAMS', 'inst.copy_network=1') =~ /inst\.copy_network=([01])/;
```

Comment:
the default value should be the one that we don't need write here and you don't need a comment neither, the code would be self-explanatory. We just need to check by `inst.copy_network` regardless of the value, if it is set, it should be set to `0` otherwise is a QE setup problem.

Reply:
Another issue that I see to do things `step-by-step` is that you need to cover the interactive somehow here and probably you will need to extend the condition reading from agama test option. It might be easier just to have `INST_COPY_NETWORK=1` and do some small modification to boot_agama.pm, wdyt?

Reply:
it makes no sense that the interactive path use a variable with a value that only is present in unattended installations to decide the logic.